### ISSUE TITLE
redirectToPath is ignored via social login

### ISSUE THREAD

**Issue by gusfune** [supertokens/supertokens-auth-react#240]
I just noticed that social login is not redirecting according to `redirectToPath` parameter.

We are using this exact implementation with NextJS (latest): https://supertokens.io/docs/thirdpartyemailpassword/common-customizations/handling-signin-success - refer to item no. 1.

With email and password, it redirects, but with social login (FB and Google), it doesn't.
I recorded a short video showing up: https://www.loom.com/share/93465728ba4d460191169a34e2fbf346?from_recorder=1
Here is on our front-end init, we didn't modify any of it:

```
ThirdPartyEmailPasswordReact.init({
        getRedirectionURL: async (context) => {
          if (context.action === "SUCCESS") {
            return context.redirectToPath === undefined
              ? "/"
              : context.redirectToPath
          }
        },
[...]
```

From what I see, it seems when the social login callback, it removes the redicrectToPath param, which defaults the redirect back to the home page.

**Comment by rishabhpoddar** [comment]
@gusfune this issue has been fixed in v0.10.1. Thanks for pointing it out.

**Comment by gusfune** [comment]
Awesome! Thanks for the quick response.
